src/test/java/weizberg/rijks/RijksServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package weizberg.rijks;
+
+import com.andrewoid.ApiKey;
+import org.junit.jupiter.api.Test;
+import weizberg.rijks.json.ArtObjects;
+import weizberg.rijks.json.ArtObjectsCollection;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+public class RijksServiceTest {
+
+    @Test
+    public void pageNumber() {
+        //given
+        ApiKey apiKey = new ApiKey();
+        String keyString = apiKey.get();
+        RijksService service = new RijksServiceFactory().getService();
+
+         //when
+         ArtObjects artObjects = service.pageNumber(
+                keyString,
+                70
+        ).blockingGet();
+
+         //then
+        assertNotEquals(0, artObjects.title);
```

Comment:
Title is a String. Of course it's not going to be equal to 0. 

intelliJ probably warns you about this.